### ISSUE TITLE
historical order book

### ISSUE THREAD

**Issue by captainbanan** [Polymarket/py-clob-client#75]
is it possible to get the state of the order book in the past? for example, can I get the best bid and ask in some market using a timestamp?

thank you in advance

**Comment by Carter4242** [comment]
@captainbanan No, I don't think that's possible. As far as I'm aware, there is no public database of bids/asks or unfilled order history.

**Comment by saguillo2000** [comment]
@Carter4242 in the documentation there is this method:

print(
    client.get_order_book(
        "71321045679252212594626385532706912750332728571942532289631379312455583992563"
    )
)

I tried with the example posted and other token_ids and seems to not being working, then you can only get the current order book without the option to recover previous bids?
